src/CmsKitDemo/Services/ImageGalleryAppService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using CmsKitDemo.Entities;
+using CmsKitDemo.Permissions;
+using CmsKitDemo.Services.Dtos;
+using Microsoft.EntityFrameworkCore;
+using System.Linq.Dynamic.Core;
+using Volo.Abp.Application.Dtos;
+using Volo.Abp.Application.Services;
+using Volo.Abp.Domain.Repositories;
+using Volo.CmsKit.Comments;
+using Volo.CmsKit.Reactions;
+
+namespace CmsKitDemo.Services
+{
+    public class ImageGalleryAppService :
+        CrudAppService<GalleryImage, GalleryImageDto, Guid, PagedAndSortedResultRequestDto, CreateUpdateGalleryImageDto, CreateUpdateGalleryImageDto>,
+        IImageGalleryAppService
+    {
+        public ImageGalleryAppService(IRepository<GalleryImage, Guid> repository) : base(repository)
+        {
+            CreatePolicyName = CmsKitDemoPermissions.GalleryImage.Create;
+            UpdatePolicyName = CmsKitDemoPermissions.GalleryImage.Update;
+            DeletePolicyName = CmsKitDemoPermissions.GalleryImage.Delete;
+        }
+
+        public async Task<List<GalleryImageWithDetailsDto>> GetDetailedListAsync()
+        {
+            var dbContext = await Repository.GetDbContextAsync();
```

Comment:
I'm not sure using DbContext in AppService

Reply:
Since it's no-layer app, it's ok ✅

Reply:
I wanted to keep it simple and therefore didn't create a custom repository. 👍